exercises/dominoes/domino.rb

```diff
@@ -0,0 +1,16 @@
+class Domino
+  attr_reader :left, :right
+
+  def self.deep_copy(domino)
+    Domino.new(domino.left, domino.right)
+  end
```

Comment:
Is this the same as https://ruby-doc.org/core-2.3.1/Object.html#method-i-clone? If so, can we use the Ruby method?